lib/puppet/defaults.rb

```diff
@@ -1719,7 +1726,7 @@ def self.initialize_default_settings!(settings)
       and does not need to horizontally scale.",
     },
     :ca_port => {
-      :default    => "$masterport",
+      :default    => "$serverport",
```

Comment:
It's amusing to me that these have apparently always been strings while the base settings were ints. I guess we've coded around that in general. Agreed it would be nice to unify these with your new `:port` type later!